doc/ref/chap-special-var.md

```diff
@@ -379,6 +379,15 @@ Override the default OSH history location.
 
 Override the default YSH history location.
 
+## Interactive
+
+### OILS_COMP_UI
+
+Set which completion UI to use. Set to `minimal` for a UI that approximates the
+default behavior of readline. Set to `nice` for a UI with a fancy pager and a
```

Comment:
I would say "GNU readline" because some people might not know what "readline" is

I think it will nicer to read with a bulleted list, like

- `minimal` - ...
- `nice` - ...

Reply:
👍 done